### DISCUSSION TITLE
`listSearchableFields` should be used when searching for relations as well

### DISCUSSION THREAD

**Discussion by luca-nardelli** [payloadcms/payload#2935] [Feature Requests & Ideas]
Hi all! We have some collections where we have multiple fields set in `listSearchableFields`. When we search in the regular list view, those fields are correctly used for search, however when we are searching on the same collection but in the context of a relation (i.e. the collection with listSearchableFields is the target of the `relationTo` property of the field) only the `useAsTitle` field is used for search.

Shouldn't we use listSearchableFields here as well?

**Comment by DanRibbens** [comment]
That is a good idea. I can't think of any reason why not.

Are you interested in contributing a PR?

**Reply by luca-nardelli** [reply]
Definitely, however I don't know where exactly I need to make the change. This morning I tried having a look at the getResults function https://github.com/payloadcms/payload/blob/0112f4c4ab1d3dc3adf31fdef644c6e416f33d46/src/admin/components/forms/field-types/Relationship/index.tsx#L102

by changing this part
```ts
            if (searchArg) {
              query.where.and.push({
                [fieldToSearch]: {
                  like: searchArg,
                },
              });
            }
```

But it looked like it didn't do anything. Do you have some pointers?
